torch_geometric/__init__.py

```diff
@@ -1,23 +1,23 @@
 from types import ModuleType
 from importlib import import_module
 
-from .seed import seed_everything
-from .debug import is_debug_enabled, debug, set_debug
-
 import torch_geometric.data
 import torch_geometric.loader
 import torch_geometric.transforms
 import torch_geometric.utils
 import torch_geometric.profile
 
+from .seed import seed_everything
```

Comment:
you could consider using `isort` to reorganize imports automatically and add the `flake8-isort` plugin as an extra requirement for the flake8 action

Reply:
I'll send a short PR demonstrating